activities/ColorMyWorld.activity/js/activity.js

```diff
@@ -85,3 +85,16 @@ define([
 		colormyworld.change_areaCB(true,'World');
 	});
 });
+		
+document.getElementById("fullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 0;
+			document.getElementById("canvas").style.top = "0px";
+			document.getElementById("unfullscreen-button").style.visibility = "visible";
+			resizeCanvas();
+		});
+document.getElementById("unfullscreen-button").addEventListener('click', function() {
```

Comment:
Please can you ensure that the indentation of the document.getElementById.... follows the indentation style of the rest of the file.

Reply:
Done